challenges/array-reverse/arrayReverse/array-reverse.js

```diff
@@ -0,0 +1,13 @@
+const a = ['rat', 'pig', 'dog', 'rooster', 'monkey', 'goat', 'horse', 'snake', 'dragon', 'rabbit', 'tiger', 'ox']
+const b = [2020,2019,2018,2017,2016,2015,2014,2013,2012,2011,2010,2009]
+
+function reverseArray(arr) {
+    let newArr = [];
+    for(let i = arr.length - 1; i >= 0; i--) {
+        newArr.push(arr[i]);
```

Comment:
not allowed to use push. think about where(the index of the array) the numbers are coming from and where they need to go. cant use any array method besides .length